library/Octane/Utility/Parser.hs

```diff
@@ -132,7 +132,25 @@ getStream replay = do
              Just (Property.PropertyInt int) -> int & #content & Int32.fromInt32
              _ -> 0)
   trace (Printf.printf "Getting %d frame(s)" numFrames)
-  replay & extractContext & getFrames 0 numFrames & fmap snd
+  let context = extractContext replay
+  context & #classMap & Map.toAscList &
+    map (\(k, v) -> show k ++ "\t" ++ show v) &
+    ("CLASS MAP" :) &
+    unlines &
+    trace
+  context & #classPropertyMap & IntMap.toAscList &
+    map
+      (\(k1, v1) ->
+         show k1 ++
+         "\n" ++
+         (v1 & IntMap.toAscList &
+          map (\(k2, v2) -> "\t" ++ show k2 ++ "\t" ++ show v2) &
+          unlines)) &
+    ("CLASS PROPERTY MAP" :) &
+    unlines &
+    trace
```

Comment:
It may not be necessary to print these out. On the other hand, it would be useful to debug problems like #37. I guess I might as well leave it in.